pyramid_learning_journal/pyramid_learning_journal/views/default.py

```diff
@@ -1,39 +1,26 @@
-    {'id': 2, 'title': 'Journal Entry 3', 'date': 'May 02, 2017', 'body': 'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.'},
-    {'id': 3, 'title': 'Journal Entry 4', 'date': 'May 05, 2017', 'body': 'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.'},
-    {'id': 4, 'title': 'Journal Entry 5', 'date': 'May 09, 2017', 'body': 'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.'},
-    {'id': 5, 'title': 'Journal Entry 6', 'date': 'May 12, 2017', 'body': 'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.'}
-]
+from pyramid_learning_journal.models import Journal
 
 
 @view_config(route_name='home', renderer='../templates/index.jinja2')
 def home_view(request):
     """View for the home route."""
+    session = request.dbsession
+    all_journals = session.query(Journal).all()
     return {
         'page': 'Home',
-        'journals': JOURNALS
+        'journals': all_journals
     }
 
 
 @view_config(route_name='detail', renderer="../templates/post.jinja2")
 def detail_view(request):
     """View for the journal route."""
     the_id = int(request.matchdict['id'])
-
-    journal = None
-    for item in JOURNALS:
-        if item['id'] == the_id:
-            journal = item
-            break
-
-    # journal = list(filter(lambda item: item['id'] == the_id, JOURNALS))
-
+    session = request.dbsession
+    journal = session.query(Journal).get(the_id)
```

Comment:
Missing add and edit views. They're only supposed to return empty dictionaries right now, but they should still be here.